src/logger/src/logger.rs

```diff
@@ -526,6 +530,40 @@ mod tests {
         assert_eq!(l.show_level(), true);
     }
 
+    #[test]
+    fn test_write_log() {
+        const TEST_HEADER: &str = "test_log";
+        const TEST_STR: &str = "testing flushing";
+        const TEST_FILE: &str = "./test.txt";
```

Comment:
Is it at all possible to have the buffered writer write to the vector directly instead of using a file? If not, I think this path should move to `/tmp` and if we could clarify the file name as well. My vote would be for `/tmp/firecracker-unit-tests-.log`.

Reply:
I believe I need to use a [`std::fs::File`](https://doc.rust-lang.org/stable/std/fs/struct.File.html) in this instance since when using an in-memory [`std::io::Cursor`](https://doc.rust-lang.org/std/io/struct.Cursor.html) since read/write times are so fast even when using a [`BufWriter`](https://doc.rust-lang.org/std/io/struct.BufWriter.html) it will not buffer writes. I agree `/tmp` would be a good idea, I will make that change.